README.md

```diff
@@ -265,14 +265,14 @@ node src/bin/facilitator.js progressRedeem path_to_config.json messageHash
   
 ## Setup OpenST
 
-Prerequisite: `organization` and `eip20Token` contracts should have been deployed.
+Prerequisite: `eip20Token` contract should have been deployed.
     
 ```bash
 # Help:
 node ./src/bin/openst.js --help
     
 # Setup OpenST:
-node ./src/bin/openst.js config.json organization eip20Token
+node ./src/bin/openst.js openst config.json eip20Token
```

Comment:
Why removed the organization address?

Reply:
It can be fetched from the config if we have based on `eip20Token` address.

Reply:
Here I think organization is an important argument for TokenRules deployment while setup of OpenST. I feel we should keep it. We can take feedback from others cc: @benjaminbollen @sarvesh-ost what do you think?

Reply:
we shouldnt assume that the organization for eip20token and tokenrules is the same - even if we currently deploy it as such